tools/torch_quant/README.md

```diff
@@ -0,0 +1,61 @@
+# Quantization Toolkit for PyTorch Models
+
+*under development*
+
+## 1. Quantization Basic Concepts
+
+Deep learning model quantization is the process of reducing the precision of the weights
+and activations of a neural network model in order to reduce the memory footprint
+and computational complexity of the model. A common way to perform quantization is
+to represent the weights and activations using fewer bits than the original model,
+typically using 8 or fewer bits.
+
+A typical workflow of quantization is as following:
+- Step 0: Pre-train model
```

Comment:
the ptq step is not memtioned

Reply:
Updated readme, and it's more accurate to put calibration before qat.